lms/static/sass/shared-v2/_help-tab.scss

```diff
@@ -0,0 +1,77 @@
+.help-tab {
```

Comment:
Nit (which you can clean up later): use four spaces for indentation in new sass files.